app/models/utils/nav/customization_parser.rb

```diff
@@ -5,10 +5,45 @@ module Nav
     # Parses the top menu customization setting
     class CustomizationParser
       attr_accessor :markdown
+      attr_accessor :customizations
 
       def initialize(markdown)
         # a nil input is stored as an empty string
         self.markdown = markdown || ""
+        self.customizations = parse
+        @done = []
+      end
+
+      def filter_item(name:, path:, permitted:, icon:)
+        # check to see if this item has been customized
+        result = {}
+        result[:name] = name
+        translated_name = I18n.t("nav_links.main.#{name}")
+        if @customizations[translated_name]
+          result[:path] = @customizations[translated_name]
+          @done << translated_name
```

Comment:
Instead of this `@done` variable, couldn't you just delete this one from customizations? Simpler that way?

Reply:
simpler, esp. since it makes the next item easier.